.github/workflows/copy-completed-notebooks.yml

```
@@ -0,0 +1,109 @@
+name: Copy completed HTML notebooks to training website repository
+# Copy completed HTML notebooks from the training-modules repo to this repo
+# This action will open a pull request containing completed HTML notebooks
+
+# This is a manually triggered workflow that takes two inputs:
+#  # The name of the workshop being taught
+#  # The training-modules repo tag to copy notebooks from
+on:
+  workflow_dispatch:
+    inputs:
+      training-module:
+        type: choice
+        description: Use the dropdown menu to select which training workshop module this website is being created for.
+        options:
+          - "Introduction to R and Tidyverse"
+          - "Introduction to single-cell RNA-seq"
+          - "Advanced single-cell RNA-seq"
+          - "Bulk RNA-seq"
+        required: true
+      training-modules-tag:
+        type: string
+        description: The `training-modules` repo tag to copy completed notebooks from, e.g. 2023-june
+        default: "master"
+        required: true
+
+jobs:
+  file-completed-notebooks-pr:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout training-modules repository
+        uses: actions/checkout@v3
+        with:
+          repository: AlexsLemonade/training-modules
+          ref: ${{ github.event.inputs.training-modules-tag }}
+          path: training-modules
+
+      - name: Checkout this repository
+        uses: actions/checkout@v3
+        with:
+          path: website
+
+      - name: Configure git
+        run: |
+          git config --global user.email "actions@github.com"
+          git config --global user.name "GitHub Actions"
+
+      - name: Copy HTML completed notebooks from training-modules
+        shell: bash
+        run: |
+          # The completed notebooks to copy will depend on which workshop module this is.
+          # For each module, the directory of notebooks to copy are:
+          #  Intro R: intro_to_R_tidyverse/
+          #  Intro scRNA-seq: intro_to_R_tidyverse/ and scRNA-seq/
+          #  Advanced scRNA-seq: advanced-scRNA-seq/
+          #  bulk RNA-seq: RNA-seq/
+
+          # Path to copy notebooks to
+          target_path=website/completed-notebooks/
+
+          # Create array of directories from which notebooks should be copied
+          case "${{ github.event.inputs.training-module }}" in
+
+            "Introduction to R and Tidyverse")
+              modules=("intro-to-R-tidyverse")
+              ;;
+
+            "Introduction to single-cell RNA-seq")
+              modules=("intro-to-R-tidyverse" "scRNA-seq")
+              ;;
+
+            "Advanced single-cell RNA-seq")
+              modules=("scRNA-seq-advanced")
+              ;;
+
+            "Bulk RNA-seq")
+              modules=("RNA-seq")
+              ;;
+          esac
+
+          # Copy all completed notebooks from the given directories
+          for module in ${modules[@]}; do
+            cp training-modules/${module}/[0-9]*.html ${target_path}
```

I think we still want to keep the modules separated? 
```suggestion
            cp training-modules/${module}/[0-9]*.html ${target_path}/${module}
```

---

yeah, seems reasonable